src/docs/release/breaking-changes/scrollable_alert_dialog.md

```diff
@@ -0,0 +1,160 @@
+        child: DefaultTextStyle(
+          style: titleTextStyle ?? dialogTheme.titleTextStyle ?? theme.textTheme.title,
+          child: Semantics(
+          child: title,
+          namesRoute: true,
+          container: true,
+          ),
+        ),
+      ),
+    if (content != null)
+      Flexible(
+        child: Padding(
+        padding: contentPadding,
+        child: DefaultTextStyle(
+          style: contentTextStyle ?? dialogTheme.contentTextStyle ?? theme.textTheme.subhead,
+          child: content,
+        ),
+      ),
+    ),
+    // ...
+  ],
+);
+```
+
+The proposed change would basically wrap both widgets in a
+SingleChildScrollView above the button bar, making both
+widgets part of the same scrollable while exposing the
+button bar at the bottom of the dialog:
+
+```dart
+Column(
+  mainAxisSize: MainAxisSize.min,
+  crossAxisAlignment: CrossAxisAlignment.stretch,
+  children: <Widget>[
+    if (title != null || content != null)
+      SingleChildScrollView(
+        child: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+         children: <Widget>[
+           if (title != null)
+             titleWidget,
+             if (content != null)
+             contentWidget,
+         ],
+       ),
+     ),
+   // ...
+  ],
+),
+```
+
+## Migration guide
+
+### Potential breaking changes
+
+1. Semantics tests may fail because of the addition of a SingleChildScrollView.
+
+Upon manually testing Talkback and VoiceOver, they still behave correctly and
+the same.
+
+However, this change might have caused semantics golden diffs in some tests
+since the SingleChildScrollView now nests both the title and content widgets.
+Some Flutter projects have taken to creating semantics tests by taking goldens
+of semantics nodes using in Flutter's debug build.
+
+Any semantics golden updates that reflect the scrolling container addition
+is expected and these diffs should be safe to accept.
+
+Sample resulting Semantics tree:
+
+```
+flutter:        ├─SemanticsNode#30 <-- SingleChildScrollView
+flutter:          │ flags: hasImplicitScrolling
+flutter:          │ scrollExtentMin: 0.0
+flutter:          │ scrollPosition: 0.0
+flutter:          │ scrollExtentMax: 0.0
+flutter:          │
+flutter:          ├─SemanticsNode#31 <-- title
+flutter:          │   flags: namesRoute
+flutter:          │   label: "Hello"
+flutter:          │
+flutter:          └─SemanticsNode#32 <-- contents
+flutter:              label: "Huge content"
+```
+
+2. This change may result in layout changes because of the scroll view.
+
+If the dialog was already overflowing, this change would correct the problem
+and this layout change is expected.
+
+A nested SingleChildScrollView in `AlertDialog.content` seems to still work
+properly if left in, but it should probably be removed if unintended since
+it might cause confusion.
+
+### Two-phase update:
+
+This change will be conducted in two phases.
```

Comment:
the migration guide is for people who are migrating AFTER we have landed the multiple phases (the multiple phases are what you do to make your own work of migrating people manageable, it's not a long-term thing, most people don't see it).

Reply:
Does this mean we only outline the multiple phases on the design doc? 

What is the intent of outlining the breaking changes on the flutter website? Does it serve to provide context for the change and a migration guide for our users?